src/Compilers/CSharp/Test/Semantic/Semantics/NullableContextTests.cs

```diff
@@ -496,7 +517,8 @@ public static IEnumerable<object[]> AnalyzeMethodsInEnabledContextOnly_01_Data()
         }
 
         [Theory]
-        [MemberData(nameof(AnalyzeMethodsInEnabledContextOnly_01_Data))]
+        [MemberData(nameof(AnalyzeMethodsInEnabledContextOnly_01_Data1))]
+        [MemberData(nameof(AnalyzeMethodsInEnabledContextOnly_01_Data2))]
```

Comment:
Does this address the issue? Don't we still have the same number of test results per test?

Reply:
@cston I'm waiting for CI to check. But I think while it's the same number per *test*, it's not the same number per *test case*. The warning is about "subresults in *test case*". But CI will say its decision :smile:

Reply:
If the same issue exists, consider having two tests that share a common helper in each case.

```csharp
[MemberData(nameof(AnalyzeMethodsInEnabledContextOnly_01_Data1)]
public void AnalyzeMethodsInEnabledContextOnly_01A(NullableContextOptions? projectContext, NullableDirectives classDirectives, NullableDirectives methodDirectives)
{
    AnalyzeMethodsInEnabledContextOnly_01_Execute(projectContext, classDirectives, methodDirectives);
}

[MemberData(nameof(AnalyzeMethodsInEnabledContextOnly_01_Data2)]
public void AnalyzeMethodsInEnabledContextOnly_01B(NullableContextOptions? projectContext, NullableDirectives classDirectives, NullableDirectives methodDirectives)
{
    AnalyzeMethodsInEnabledContextOnly_01_Execute(projectContext, classDirectives, methodDirectives);
}

private void AnalyzeMethodsInEnabledContextOnly_01_Execute(NullableContextOptions? projectContext, NullableDirectives classDirectives, NullableDirectives methodDirectives)
{
    ...
}
```

---
In reply to: [589068480](https://github.com/dotnet/roslyn/pull/51730#discussion_r589068480) [](ancestors = 589068480)

Reply:
@cston You're correct, the warnings are still there 😕 

![image](https://user-images.githubusercontent.com/31348972/110250546-788f4d00-7f84-11eb-852a-bacaabe6c2fb.png)

I don't have a good solution in mind. Duplicating the exact same test doesn't seem to be a good option. I'll close and let someone else come with a good solution.

Reply:
@cston Oh, I had the PR tab open and it didn't show your previous reply when I responded. I'll reopen and apply your change. Thanks.